docs/reference/modules/ROOT/pages/http.adoc

```diff
@@ -73,183 +73,276 @@ If the node was started on `localhost:3000`, you can connect to it by doing the
 include::example$src/docs/examples.clj[tags=start-http-client]
 ----
 
-[#http-api]
-== Using the HTTP API
+[#rest-api]
+== Using the REST API
 
-.API
-[#table-conversion%header,cols="d,d,d"]
-|===
-|uri|method|description
-|<<#entity, `/entity/[:key]`>>|GET|Returns an entity for a given ID and optional valid-time/transaction-time co-ordinates
-|<<#entity-tx, `/entity-tx/[:key]`>>|GET|Returns the transaction that most recently set a key
-|<<#entity, `/entity-history/[:key]`>>|GET|Returns the history of the given entity and optional valid-time/transaction-time co-ordinates
-|<<#query, `/query`>>|POST|Takes a datalog query and returns its results
-|<<#sync, `/sync`>>|GET| Wait until the Kafka consumer's lag is back to 0
-|<<#tx-log, `/tx-log`>>|GET| Returns a list of all transactions
-|<<#tx-log-post, `/tx-log`>>|POST|The "write" endpoint, to post transactions.
-|===
+All of the REST endpoints return both `application/edn` and `application/transit+json` (see transit https://github.com/cognitect/transit-clj[here]) - individual endpoints may return additional types - see the docs below. To set up your node to with the REST API, see the xref:http.adoc[HTTP Feature Module].
```

Comment:
can use '`application/transit+json`' as the link text, rather than a separate 'see ...'

Reply:
With adoc, I'm reliably informed (:smile:) that the idiom is one sentence per line

Reply:
I'd break the sentence before 'individual endpoints', too.

Also, 'see the HTTP Feature module' - you're already there :)

Reply:
in short:
```
All of the REST endpoints return both `application/edn` and xref:https://...[`application/transit+json`]
Individual endpoints may return additional types - see their docs below. 
```